powerauth-java-client-axis/src/main/java/io/getlime/security/powerauth/soap/axis/client/PowerAuthServiceClient.java

```diff
@@ -336,14 +336,33 @@ private GregorianCalendar calendarWithDate(Date date) {
     }
 
     /**
-     * Call the getActivationStatus method of the PowerAuth 3.0 Server SOAP interface.
+     * Call the getActivationStatus method of the PowerAuth 3.0 Server SOAP interface. This method should be used only
+     * to acquire the activation status for other, than RESTful API purposes. If you're implementing the PowerAuth RESTful API,
```

Comment:
This sentence is a bit confusing, because PA server also has a RESTful API next to SOAP API. I believe you are referencing the powerauth-restful-integration project, right? We'd better make this distinction.

Reply:
Fixed. Now I'm using naming from [our own documentation](https://developers.wultra.com/docs/2019.05/powerauth-crypto/Architecture-Overview) (e.g. PowerAuth standard RESTful API)